gunicorn_config.py

```diff
@@ -1,9 +1,5 @@
-# avoid MonkeyPatchWarning error/warnings
-from gevent import monkey
-
 from config.utils import running_dev
 
-monkey.patch_all(thread=False, select=False)
```

Comment:
doing this monkey patching here resolved some variants of bad gateway ... I would leave this asis

Reply:
+1 ... main concern is avoiding Bad Gateways ... testing revealed this change mitigated some of the issues in this area. It might be other mitigations elsewhere now make this not applicable.